c_api/src/json_api.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+use c2pa::{Ingredient, Manifest, Reader};
+use cawg_identity::validator::CawgValidator;
+use tokio::runtime::Runtime;
+
+use crate::{Error, Result, SignerInfo};
+
+/// Returns the version of the c2pa SDK used in this library
+pub fn sdk_version() -> String {
+    String::from(c2pa::VERSION)
```

Comment:
Note: No covered by tests here, but extensively covered by the languages that bind to the related C native code.